pkg/auth/jwt.go

```diff
@@ -110,19 +110,23 @@ func (j *JwtAuthenticator) ParseAndValidate(tokenString string) (jwt.MapClaims,
 // The keys are in a public key format and are converted to RSA Public Keys
 func (j *JwtAuthenticator) refreshJwksKeys() error {
 	oidcURL := os.Getenv(OIDCServerURL)
-
+	fmt.Println("***********InSideAuthJwt_SMAL_************")
```

Comment:
this is just printing to standard out - it's better to use a log message - and besides it should contain something informative - such as the URL